### DISCUSSION TITLE
Nueva categoría dentro del proyecto.

### DISCUSSION THREAD

**Discussion by laurolmer** [perealberto/decide-single-zambrano#47] [Ideas]
Chicos estaba pensando en que como tenemos varias tareas que se han cerrado porque no se iban a realizar al final, había pensado en agregar a nuestro proyecto una nueva columna que fuera "Not planned" o con algún nombre similar para poder hacer saber que esas issues al final no se van a realizar por diversos motivos.
¿Qué pensáis?

**Comment by perealberto** [comment]
Me parece bien. En la última clase de revisión de proyectos había un grupo que estaba haciendo justamente esto. Llamaron a esta columna del proyecto _Descarte_.

**Comment by silvglad** [comment]
Lo que dice Alberto me parece perfecto, podemos crear una columna llamada "Descartes" para estas issues y, así, mantener un registro de aquellas ideas que se han quedado en el tintero.
